Sources/ApolloCodegenLib/Glob.swift

```diff
@@ -206,7 +206,8 @@ public struct Glob {
 
     CodegenLogger.log("Evaluating \(pattern)", logLevel: .debug)
 
-    let response = glob(pattern, flags, nil, &globT)
+    let encodedPattern = pattern
```

Comment:
Any reason for the new variable assignment here?

Reply:
Oops, was doing some debugging and forgot to fix that. I'll fix it before merging.